network_simulator/statistics.py

```diff
@@ -69,8 +72,7 @@ class Statistics(object):
     Intended to be owned by the main loop to record all statistics within
```

Comment:
note: gap between classes (three lines above this one) should be only 2 lines long.